frontend-instructor/src/components/coursesCreate/CreateMilestone.jsx

```
@@ -0,0 +1,191 @@
+            </div>
+            <div className='flex space-x-2'>
+              <Link
+                to={`/module/${milestone.id}`}
+                className='px-4 py-2 bg-green-600 text-white font-medium rounded-md hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-500 focus:ring-offset-2'
+              >
+                Modules
+              </Link>
+              <button
+                onClick={() => handleEditClick(milestone)}
+                className='px-4 py-2 bg-indigo-600 text-white font-medium rounded-md hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2'
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)